module5/lessons/Week5/LinkedInActivityPD.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: page
+title: LinkedIn Activity
+---
+
+Today is all about returning to your LinkedIn. You will be polishing up your profile and engaging with the broader developer community to make sure your LinkedIn best represent you!
+
+## Careerflow Competition! 
+
+Instructions for all team members:
+
+1. Install the [Careerflow Chrome Extension](https://www.google.com/url?q=https://chrome.google.com/webstore/detail/careerflow-ai-linkedin-op/iadokddofjgcgjpjlfhngclhpmaelnli&sa=D&source=calendar&ust=1695826733869948&usg=AOvVaw03e0L95JHMmx0VCrDyAm3g)
+2. Go to your Linkedin profile and click the blue box with a “C” in it on the right hand side of the page.
+    * It will prompt you to make an account to view suggested improvements.
+3. Make the recommended changes to your profile to improve it - you have until lunch on Thursday to make changes
+4. Add up the percentages of each team member and average them.

```

Comment:
I wonder if we need to include an example here?  I don't think the math is difficult, but just the words 'percentage' and 'average' in the same sentence might be daunting.

Reply:
I like that idea of showing a quick example! Will add :)